src/sentry.ts

```diff
@@ -84,14 +84,30 @@ export default class SentryService {
 
       const sentry_user = await this.getUser();
 
-      const sentry_tags = {
+      const command_class = this.command.getClass();
+      const { args, flags, metadata } = await this.command.parse(command_class);
+
+      const default_flags = new Set(Object.entries(metadata.flags)
+        .filter(([flag_name, flag_metadata]) => flag_metadata?.setFromDefault)
+        .map(([flag_name]) => flag_name));
+
+      const non_default_flags = Object.keys({ ...flags })
+        .filter((flag_name) => !default_flags.has(flag_name));
+
+      const used_flags: { [key: string]: string | undefined } = {};
```

Comment:
What's the case where the flag value is undefined? Does a tag with an undefined value actually still show up in sentry?

Reply:
updated to only add optional sentry_user tags if present instead of using unknown type